sdk/src/main/scala/com/horizen/certificatesubmitter/CertificateSubmitter.scala

```diff
@@ -433,6 +435,16 @@ class CertificateSubmitter[T <: CertificateData](settings: SidechainSettings,
     }
   }
 
+  private def keyRotationMessageToSign: Receive = {
```

Comment:
considering my comment in SidechainSubmitterApiRoute class, revert the changes in this file.
Same for CertificateSubmitterTest file.

Reply:
Done